File: java/mix/C01FibType2.java
public class C01FibType2 {
    // Variables name change
    public static int fib(int n) {
        int x = 0, y = 1;
        int k = 0;
        while (k < n) {
            int t = x;
            x = y;
            y = t + y;
            k++;
        }
        return x;
    }

    public static void main(String[] args) {
        System.out.println(fib(10));
    }
}

File: dataset/javascript_mix/C01FibType2.js
// Variables name change
function fib(n) {
    let x = 0, y = 1;
    let k = 0;
    while (k < n) {
        const t = x;
        x = y;
        y = t + y;
        k++;
    }
    return x;
}

console.log(fib(10));

module.exports = { fib };
